tests/bismark_emseq_clip_r1.nf.test

```diff
@@ -3,6 +3,7 @@ nextflow_pipeline {
     name "Test Workflow main.nf"
     script "../main.nf"
     config "./nextflow.config"
+    tag "cpu"
```

Comment:
I don't like this approach too much. Easier if we filter out gpu tags as the exception.

Reply:
Agreeing with @mashehu but we don't have a nice solution for this now I'm afraid

Reply:
And as long at this is pipeline specific and not at the modules level, I'm not too fussed about it here